app/components/forms/orders/order-form.js

```diff
@@ -38,6 +38,14 @@ export default Component.extend(FormMixin, {
   }),
   sameAsBuyer: false,
 
+  isBillingInfoNeeded: computed('event', 'data.isBillingEnabled', function() {
+    return this.event.isBillingInfoMandatory || this.data.isBillingEnabled;
+  }),
+
+  isBillingNotMandatory: computed('event', function() {
+    return !this.event.isBillingInfoMandatory;
+  }),
```

Comment:
Why can't you just use not in the template

Reply:
For or and not, I will need to use ember arithmentic helper add-on which cost more then the js checking

Reply:
Not talking about or. I'm talking about `not`

Reply:
done the not